src/commands/purge.ts

```diff
@@ -43,6 +41,7 @@ const purgeModule: CommandType = {
     const channel = interaction.channel as TextChannel | ThreadChannel;
 
     // bulk delete (n + 1) number of messages
+    await interaction.reply(`Purging ${n} messages...`);
```

Comment:
personally, i prefer it with no message from the bot but i want to hear your thoughts on this